python/pyspark/ml/base.py

```diff
@@ -47,6 +74,24 @@ def _fit(self, dataset):
         """
         raise NotImplementedError()
 
+    @since("2.3.0")
+    def fitMultiple(self, dataset, params):
+        """
+        Fits a model to the input dataset for each param map in params.
+
+        :param dataset: input dataset, which is an instance of :py:class:`pyspark.sql.DataFrame`.
+        :param params: A list/tuple of param maps.
```

Comment:
Let's explicitly check that this is a list or tuple and throw a good error message if not.

Reply:
I changed the docstring to `Sequence` instead of `list/tuple`, is that ok? Do you want to explicitly restrict the input to be a list or tuple?

Reply:
Is there another Sequence type this could be other than list or tuple?